charts/outline/Chart.yaml

```diff
@@ -4,7 +4,7 @@ maintainers:
     url: https://kubito.dev
 apiVersion: v2
 appVersion: 0.80.2
-version: 1.1.0
+version: 1.2.0
```

Comment:
It's really a patch, so please change it to `1.1.1`. I really need to automate this, sorry!